templates/login.html

```diff
@@ -86,12 +86,16 @@ <h1 class="lead">
   
   </div>
 
-  <div class="footer col-md-12">
+    <div class="footer col-md-12">
     <p>
       This service is run transparently from
-      <a href="https://github.com/2i2c-org/pilot-hubs">
-        github.com/2i2c-org/pilot-hubs
+      <a href="https://github.com/2i2c-org/infrastructure">
+        github.com/2i2c-org/infrastructure
```

Comment:
The ".../pilot-hubs" URL is out of date; changed to point to the correct repo where 2i2c manages ghg.center and other hubs.